README.md

```diff
@@ -1,30 +1,100 @@
+This is the main repository for the PegNet application.
 
-4. Replace the Oracle.APILayerKey key with your API Key
+Pegged tokens reflect real market assets such as currencies, precious metals, cryptocurrencies, commodities, etc. The conversion rates on PegNet are determined by a decentralized set of miners who submit values based on current market data. These values are then graded and recorded in the Factom blockchain. 
 
-5. Edit the Miner.IdentityChain to add a unique name like:
+The draft proposal paper is available [here](https://docs.google.com/document/d/1yv1UaOXjJLEYOvPUT_a8RowRqPX_ofBTJuPHmq6mQGQ).
 
-6. IdentityChain=prototype,BigMiner
+For any questions, troubleshooting or further information head to [discord](https://discord.gg/V6T7mCW).
 
-7. Right now you can donate to the cause, and leave the pPNT address alone. We will had a utility to generate your own PNT address from one of your FCT addresses in your wallet
+All mining is currently done on Testnet.
 
-7.5 Run a local factomd instance, and run factom-walletd.  This will give you a factom node to run against, and a wallet to write entries with.  There is an Open Node configuration, and we will add instructions for that soon.
+## Mining
 
-8. Buy some Entry Credits and fund one of your EC addresses in your factom-walletd wallet
+#### Requirements
 
-9. Download the image checked into the PegNetDistribution repository
+* Pegnet binary from the [releases page](https://github.com/pegnet/pegnet/releases)
+* Factom binaries from their [distribution page](https://github.com/FactomProject/distribution/releases)
+* Factom Address that is funded or Entry Credits which can be purchased directly [here](https://shop.factom.com/) or [here](https://ec.de-facto.pro/).
+
+#### Setup
+
+Create a `.pegnet` folder inside your home directory. Copy the `defaultconfig.ini` [file](https://raw.githubusercontent.com/pegnet/pegnet/master/defaultconfig.ini) there. 
+
+On Windows this is your `%USERPROFILE%` folder
+
+Linux example:
+```bash
+mkdir ~/.pegnet
+wget https://raw.githubusercontent.com/pegnet/pegnet/master/defaultconfig.ini -P ~/.pegnet/
+```
+
+* Sign up for an API Key from https://currencylayer.com, replace APILayerKey in the config with your own
+
+* Replace either ECAddress or FCTAddress with your own
+* Modify the IdentityChain name to one of your choosing.
+* Have a factomd node running on mainnet.
+* Have factom-walletd open
+* Start Pegnet
+
+On first startup there will be some delay while the bytemap is shuffled. It will only start mining at the beginning of a block also.
+
+## Developer Sandbox Setup
+
+Docker guide can be found [here](https://github.com/pegnet/pegnet/blob/develop/Docker.md) for an automated solution.
+
+### Manual Environment Setup
+
+Factom binaries
+
+https://github.com/FactomProject/distribution/releases
+
+
+```bash
+# In first terminal
+factomd --count=5 --net=alot+ --blktime=120 --enablenet=false --network=LOCAL # change blktime to whatever suits
```

Comment:
why run a simulation with 5 nodes for a sandbox?

Reply:
Copied from the docker-compose setup I use a custom net in hindsight, perhaps something like that

Reply:
could also link them to https://docs.factomprotocol.org/start/developer-sandbox-setup-guide in order to avoid redundant explainers, though that might be overkill for some.

maybe a link to the sandbox guide and a "default" command they can use if they don't want to deal with it? something simple like
`factomd -blktime=120 -network=LOCAL`